python/pyspark/rdd.py

```diff
@@ -244,13 +244,13 @@ def persist(self, storageLevel=StorageLevel.MEMORY_ONLY):
         self._jrdd.persist(javaStorageLevel)
         return self
 
-    def unpersist(self):
+    def unpersist(self, blocking=False):
```

Comment:
Not sure if we need to annotate this as since 3.0 somehow? 
I'd also be OK just not adding this, but it makes the APIs consistent.

Reply:
cc: @HyukjinKwon

Reply:
To be strict, we could add like:

```
        .. versionchanged:: 3.0
           Added optional `blocking` to specify blocking Whether to block until all
           blocks are deleted.
```